app/controllers/admin/topics_controller.rb

```diff
@@ -312,6 +312,43 @@ def assign_team
     end
   end
 
+  def split_topic
+    parent_topic = Topic.find(params[:topic_id])
+    parent_post = Post.find(params[:post_id])
+
+    @topic = Topic.new(
+      name: t('new_discussion_topic_title', original_topic: parent_topic.name, default: "Split from #{parent_topic.name}"),
+      user: parent_topic.user,
+      forum_id: 1,
+    )
+
+    @posts = @topic.posts
+
+    if @topic.save
+      @posts.create(
+        body: parent_post.body,
+        user: parent_post.user,
+        kind: 'first',
+        screenshots: parent_post.screenshots,
+        attachments: parent_post.attachments,
+      )
+
+      parent_topic.posts.create(
+        body: t('new_discussion_post', default: 'A new discussion was created from this one'),
```

Comment:
I think this would be more usable if we changed it to include the new discussion ID number.  So it would default to "Discussion %{topic_id} was created from this."